src/bindings/python/tests/test_graph/test_basic.py

```diff
@@ -61,6 +61,7 @@ def test_graph_api():
     assert list(model.get_output_shape(0)) == [2, 2]
     assert (model.get_parameters()[1].get_partial_shape()) == PartialShape([3, 4, 5])
     assert len(model.get_parameters()) == 3
+    assert model.get_parameters().index(parameter_c) == 2
```

Comment:
There is also `model.get_parameter_index(parameter)` available in Python and C++ OV API.
https://github.com/openvinotoolkit/openvino/blob/68f7d336e5a5ffaec88344b7f2eb36738b0306a4/src/bindings/python/src/pyopenvino/graph/model.cpp#L1094-L1099